boring-sys/Cargo.toml

```
@@ -27,5 +27,5 @@ include = [
 ]
 
 [build-dependencies]
-bindgen = "0.57"
+bindgen = "0.58"
```

Can you update to 0.59 instead? That's the latest version.
```suggestion
bindgen = "0.59"
```